UPGRADE-2.0.md

```diff
@@ -4,3 +4,5 @@ Upgrade from 1.x to 2.0
 ### BaseEnum class update
 
 * `getConstants`, `isValidName` and `isValidValue` are now final.
+* `Greg0ire\Enum\BaseEnum::isValidName` method does not have `$strict` argument anymore.
```

Comment:
Please add missing words : **The** `Greg0ire\Enum\BaseEnum::isValidName` method does not have **a** `$strict` argument anymore.